api/src/test/java/org/pmiops/workbench/db/dao/UserServiceTest.java

```diff
@@ -147,7 +145,7 @@ public void setUp() {
     providedWorkbenchConfig.termsOfService.latestAouVersion = 5; // arbitrary
 
     // key UserService logic depends on the existence of the Registered Tier
-    registeredTier = accessTierDao.save(createRegisteredTier());
+    accessTierDao.save(createRegisteredTier());
```

Comment:
this is a harmless change (and arguably a cleanup) in order to force api-unit-tests execution